api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -322,8 +322,16 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	
 	/**
 	 * @see org.openmrs.api.ConceptService#getConceptsByMapping(java.lang.String, java.lang.String)
+	 * This method has been deprecated in favor of getConceptIdsByMapping, which
+	 * is used by the ConceptService to leverage caching of results
 	 */
+	@Deprecated
 	public List<Concept> getConceptsByMapping(String code, String sourceName, boolean includeRetired);
+
+	/**
+	 * @see org.openmrs.api.ConceptService#getConceptIdsByMapping(String, String, boolean)
+	 */
```

Comment:
I would also add a since annotation for the version when this method was introduced.